src/fides/api/api/v1/endpoints/privacy_experience_endpoints.py

```diff
@@ -264,6 +268,9 @@ def embed_experience_details(
     has_tcf_contents: bool = any(
         getattr(tcf_contents, component) for component in TCF_COMPONENT_MAPPING
     )
+    if has_tcf_contents:
+        privacy_experience.gvl = load_gvl()
```

Comment:
😞 this is way too slow. I'm not reloading from the file each time.  Is it something about serializing all of this in FastAPI?

Reply:
oh it might be worse in the swagger UI!

Reply:
the swagger UI does slow this down a ton, it's really bad with rendering the large payload. What does it look like with a cURL?

fwiw this is a general concern of mine with including this as an embedding object in the payload, we're bloating this endpoint with data that doesn't necessarily belong here IMO.

that being said, I think we should be able to get this to run pretty quickly, it's not _that_ much data, and outside of the swagger UI, the plus endpoint has never been noticeably slow to me. We can try comparing directly?

Reply:
when cURLing the `/privacy-experiences` endpoint with a `gvl` property populated in the response, i'm getting a response times in the range of ~100-300ms. that's certainly slower than without the `gvl` data in the response (~50ms) but that feels like it could be unavoidable overhead in serializing and sending that much response data back - the response size goes up by over 100x when including the `gvl` data (from ~30k to ~500kb in my case).

if those response times aren't acceptable for this endpoint, then i'm not sure we should be putting this data here (not suggesting that the response times are acceptable or not, i'm legitimately unsure!). if we _do_ want to embed the data here, i think we should certainly add a query param to enable/disable whether the `gvl` data is included, so that clients have the option of not including it if/when it's not needed.

Reply:
thank you Adam. Discussed internally, I'll be adding an `include_gvl` query param that is only applicable if we also have TCF contents